app/src/components/main/analytics/events/ga4Events/helpAndSupportEvents.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getBasicClickEventParameters, getBasicChooseEventParameters } from '../common/ga4Utils';
```

Comment:
Could we reuse events from app/src/components/main/analytics/events/sidebarFooterEvents.js ?

If can not, we can remove them

Reply:
We cannot reuse. The task does not say anything about deleting old events. Perhaps this is still needed.

Reply:
So, could you please provide me an example, where they can be needed. We do not have even abillity to track them.

Reply:
I found out that Google does not support old events. It was not specified in the task.